src/index.html

```diff
@@ -22,6 +22,362 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="/src/images/logo.png"
```

Comment:
The path to the logo image starts with '/src'. It's a good practice to use relative paths instead of absolute paths. This ensures that your code will still work if the folder structure of your project changes.